src/repl/scala/tools/nsc/interpreter/MemberHandlers.scala

```diff
@@ -162,36 +170,34 @@ trait MemberHandlers {
   }
 
   class TermMacroHandler(member: DefDef) extends MacroHandler(member) {
-    def notification(req: Request) = s"defined term macro $name: ${req.typeOf(name)}"
+    def notification(req: Request) = s"// defined term macro ${req.defTypeOf(name)}"
```

Comment:
I know this was here already, but it seems contrary to what we do for normal `def`s...

Reply:
I agree that I would prefer to match def, but is there any other need to signal that this is a macro?

Reply:
I don't see why. The normal printer would say `<macro> def foo...` but that's not valid scala...

Reply:
trying printing the same as def